src/commands/restore-latest.yml

```diff
@@ -0,0 +1,15 @@
+description: >
+  Restores the latest backup available of an environment to Skpr.
+
+parameters:
+  env:
+    type: string
+    default: ""
+    description: "The app environment"
+
+steps:
+  - run:
+      environment:
+        SKPR_ENV: <<parameters.env>>
```

Comment:
Needs source and target environment names

Reply:
https://github.com/skpr/skpr-orb/blob/8e5c8bf6cea3b2ecf433de0c4d53dd0d2e45f7f3/src/commands/restore.yml#L5-L12